vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -19,22 +19,15 @@
 #include "private.h"
 
 #include <gui/functions.h>
-#include <gui/state.h>
-
-#include <host/functions.h>
-#include <host/sfo.h>
-#include <host/state.h>
-
-#include <io/vfs.h>
-
-#include <util/log.h>
 
 #include <nfd.h>
 
 namespace gui {
 
+bool delete_game_file = false;
```

Comment:
so you just removed the `static` keyword? lol
the issue is not with just static
don't use globals
put everything in `GuiState` like I told you

Reply:
I can't here for one things, and master already use global few time for code, and here is only used for this file, not need est this on gui state

Reply:
Why can't you?
We use globals when it's necessary, not out of lazyness.

Reply:
i can't like ni said, is not possible for nfd

Reply:
Do it for the other 3 globals then...

Reply:
if i keep one here, i can also keep other, is not need set it on gui state, only this file using it but here i try somethings

Reply:
Leave it, I'll just merge.